### PR TITLE
README: add the quay build status

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/containers/BuildSourceImage.svg?branch=master)](https://travis-ci.org/containers/BuildSourceImage)
+[![Docker Repository on Quay](https://quay.io/repository/vbatts/bsi/status "Docker Repository on Quay")](https://quay.io/repository/vbatts/bsi)
```
Comment (rhatdan) [inline README.md]: Can we say this without "Docker"...
Container Image Repository on Quay

**Comment by vbatts** [comment]
Ah fair enough. That is copy paste. Probably a good request to the quay
folks to change their default string.

**Comment by vbatts** [comment]
fixed. PTAL

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**